packages/expo-video/android/src/main/java/expo/modules/video/records/SeekTolerance.kt

```diff
@@ -0,0 +1,28 @@
+package expo.modules.video.records
+
+import androidx.media3.common.util.UnstableApi
+import androidx.media3.common.util.Util
+import androidx.media3.exoplayer.SeekParameters
+import expo.modules.kotlin.records.Field
+import expo.modules.kotlin.records.Record
+import expo.modules.video.player.VideoPlayer
+import java.io.Serializable
+
+@UnstableApi
+class SeekTolerance(
+  @Field var toleranceBefore: Double = 0.0,
+  @Field var toleranceAfter: Double = 0.0
```

Comment:
```suggestion
  @Field var toleranceBefore = Duration.ZERO,
  @Field var toleranceAfter = Duration.ZERO
```
You can use [kotlin `Duration`](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.time/-duration/) class. It'll expect a double that represents seconds, and then you can easily convert to milliseconds.